site/src/pages/TemplatePage/TemplateSummaryPage/TemplateSummaryPage.test.tsx

```diff
@@ -50,4 +51,17 @@ describe("TemplateSummaryPage", () => {
     const dropdownButton = screen.queryByLabelText("open-dropdown")
     expect(dropdownButton).toBe(null)
   })
+  it("shows the template version warning", async () => {
+    server.use(
+      rest.get(
+        "/api/v2/templateversions/:templateVersionId",
+        async (req, res, ctx) => {
+          return res(ctx.status(200), ctx.json(MockTemplateVersion3))
+        },
+      ),
+    )
+    renderPage()
+    await screen.findByText(MockTemplate.display_name)
+    await screen.findByTestId("warning-deprecated-parameters")
+  })
```

Comment:
I removed the logic test in favor of the storybook use case but decided to leave a separate storybook for TemplateVersionWarnings. If we decided to add more warnings, it will be a storybook "album" to preview all possible variations.